### PR TITLE
[ArtemisNG] Better errors handling

### PR DESCRIPTION
**Pull request by mbounabi**
In case the coverage passed as a parameter isn't in the available coverages list

### CODE REVIEW

File: artemis/docker_orchestrator.py [hunk 1]
```diff
@@ -190,99 +190,102 @@ def launch_coverages(coverages, logs):
...
-                    else ""
-                )
-                docker_instance.write(
-                    template.render(
-                        instances=instance_render,
-                        kraken_env=kraken_env_parameters,
-                        jormun_env=jormun_env_parameters,
+        if not list_of_coverages:
+            logger.error("Couldn't find instance matching '{}'".format(coverage_to_run))
+        else:
```
Comment (mbounabi) [inline artemis/docker_orchestrator.py:195]: The diff isn't well handled. Only these 3 lines were added, the rest is indentation.

Comment (pbench) [reply artemis/docker_orchestrator.py:195]: The `coverage_to_run` variable is not defined at this point.
Maybe two different error messages depending on if `coverages` is set or not ?

File: artemis/docker_orchestrator.py [hunk 2]
```diff
@@ -184,105 +187,112 @@ def launch_coverages(coverages, logs):
...
-                list_of_coverages.extend(
-                    [x for x in data["instances"] if coverage_to_run in x]
-                )
+                coverage_to_add = [x for x in data["instances"] if coverage_to_run in x]
+                if not coverage_to_add:
+                    logger.warning("No coverage matches '{}'".format(coverage_to_run))
+                else:
+                    list_of_coverages.extend(
+                        [x for x in data["instances"] if coverage_to_run in x]
+                    )
```
Comment (pbench) [inline artemis/docker_orchestrator.py:196]:
```suggestion
                    list_of_coverages.extend(
                        coverage_to_add
                    )
```

Comment (mbounabi) [reply artemis/docker_orchestrator.py:196]: Of course! It is the whole point to store it in a variable. Silly mistake!

File: artemis/docker_orchestrator.py [hunk 3]
```diff
@@ -18,6 +18,13 @@
 logger = logging.getLogger("NG_ORCHESTRATOR")
 
 
+def check_argument_path(arg: str):
+    if arg not in config:
+        raise Exception("{} needs to be set".format(arg))
+    if not os.path.isdir(config[arg]):
+        raise Exception("{} isn't a valid path".format(config[arg]))
+
```
[the remark this answers is not in the document]
Comment (pbench) [reply artemis/docker_orchestrator.py:26]: `config` is a global

**Pull request merged**